client/blocks/reader-featured-video/index.jsx

```diff
@@ -61,14 +65,11 @@ class ReaderFeaturedVideo extends React.Component {
 	throttledUpdateVideoSize = throttle( this.updateVideoSize, 100 )
 
 	handleThumbnailClick = ( e ) => {
-		e.preventDefault();
-		this.props.onThumbnailClick();
-
-		if ( ! this.props.allowPlaying ) {
-			return false;
+		if ( this.props.allowPlaying ) {
```

Comment:
I seem to remember we needed to have the 

`this.props.onThumbnailClick();`

regardless of whether `allowPlaying` was enabled. For combined card I think, so that the full post opens when the thumb is clicked?